examples/large_models/deepspeed/bloom/custom_handler.py

```diff
@@ -0,0 +1,113 @@
+import logging
+from abc import ABC
+
+import torch
+import transformers
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ts.context import Context
+from ts.torch_handler.distributed.base_deepspeed import BaseDeepSpeedHandler
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+
+
+class TransformersSeqClassifierHandler(BaseDeepSpeedHandler, ABC):
+    """
+    Transformers handler class for sequence, token classification and question answering.
+    """
+
+    def __init__(self):
+        super(TransformersSeqClassifierHandler, self).__init__()
+        self.max_length = None
+        self.tokenizer = None
+        self.initialized = False
+
+    def initialize(self, ctx: Context):
+        self.max_length = int(ctx.model_yaml_config["handler"]["max_length"])
+        self.model = AutoModelForCausalLM.from_pretrained(
+            "bigscience/bloom-7b1", use_cache=False
```

Comment:
This is fine for an example but I'm still curious how you expect people to use DeepSpeed
1. Derive the DeepSpeed handler?
2. Copy paste this file and make appropriate changes?

I'm not sure I'm fan of having a tutorial per large model vendor and then for each one have lots of code duplication between the custom handlers

Reply:
per our offline sync up, we want to let user clearly know the steps, instead of hiding the details in deepspeed base handler. 

will add instruction in readme